src/website/app/demos/Pagination/examples/visibleRange.js

```diff
@@ -0,0 +1,49 @@
+/* @flow */
+import { Component } from 'react';
+import Pagination from '../../../../../library/Pagination';
+
+export default {
+  id: 'visible-range',
+  title: 'Visible Range',
+  description: `The width of the page navigation can be controlled with the
+\`visibleRange\` prop. The value represents the length of the range to be
+displayed between ellipses when the full range of pages is truncated. The value
+should be an odd number, by nature of the design.`,
+  scope: { Component, Pagination },
+  source: `
+    () => {
+      const visibleRange = 3
```

Comment:
Can we just inline this value in the JSX?  The indirection makes the example more complex than it needs to be.